bundles/InstallBundle/dump/install.sql

```diff
@@ -691,7 +691,7 @@ CREATE TABLE `users_workspaces_object` (
 DROP TABLE IF EXISTS `uuids`;
 CREATE TABLE `uuids` (
   `uuid` CHAR(36) NOT NULL,
-  `itemId` int(11) unsigned NOT NULL,
+  `itemId` VARCHAR(50) NOT NULL,
```

Comment:
Should be also changed in https://github.com/pimcore/pimcore/blob/11.x/bundles/UuidBundle/src/Resources/install/install.sql
Why is the table also in the InstallBundle? Should be only in UuidBundle, right?

Reply:
Good catch! yes, that should only be in UuidBundle installer.